test/interface/scoregradelbo.jl

```diff
@@ -1,5 +1,14 @@
 
-using Test
+AD_scoregradelbo_interface = if TEST_GROUP == "Enzyme"
+    [AutoEnzyme()]
+else
+    [
+        AutoForwardDiff(),
+        AutoReverseDiff(),
+        AutoZygote(),
+        AutoMooncake(; config=Mooncake.Config()),
+    ]
+end
```

Comment:
If this gets used a lot, could it be in a test utils module? Orthogonal to this PR though, can be done later.

Reply:
Yes I am planning to iron this out in a separate PR